app/src/main/java/org/oppia/app/profile/AddProfileActivityPresenter.kt

```diff
@@ -103,7 +105,11 @@ class AddProfileActivityPresenter @Inject constructor(
         return@setOnClickListener
       }
 
-      profileManagementController.addProfile(name, pin, selectedImage, allowDownloadAccess, isAdmin = false)
+      profileManagementController.addProfile(
+        name, pin, selectedImage, allowDownloadAccess, activity.intent.getIntExtra(
+          KEY_PROFILE_AVATAR_COLOR, R.color.avatar_background_1
+        ), false
```

Comment:
Nit: please keep the ``isAdmin = false`` part for the literal value. It helps document what the literal value corresponds to in the argument list.

Reply:
Updated.